tests/test_perf_analyzer_config_generator.py

```diff
@@ -287,6 +287,34 @@ def test_max_concurrency(self):
         self._run_and_test_perf_analyzer_config_generator(
             yaml_content, expected_configs, pa_cli_args)
 
+    def test_min_concurrency(self):
+        """ 
+        Test Min Concurrency: 
+            - Change min concurrency to non-default value
+        
+        Min Concurrency: 5
+        3 configs [5, 8, 16] will be generated 
+        """
+
+        # yapf: disable
+        yaml_content = convert_to_bytes("""
+            profile_models:
+                - my-model
+            """)
+        # yapf: enable
+
+        concurrencies = utils.generate_doubled_list(5, 16)
```

Comment:
I'd rather this be explicitly called out as [5,10] instead of showing what low level function the code relies on.